src/sentry/sentry_apps/api/endpoints/sentry_app_avatar.py

```diff
@@ -6,8 +6,8 @@
 from sentry.api.base import control_silo_endpoint
 from sentry.api.bases import SentryAppBaseEndpoint
 from sentry.api.bases.avatar import AvatarMixin
-from sentry.api.serializers.rest_framework.sentry_app import SentryAppAvatarSerializer
 from sentry.models.avatars.sentry_app_avatar import SentryAppAvatar
+from sentry.sentry_apps.api.serializers.sentry_app_requests import SentryAppAvatarSerializer
```

Comment:
I think it is a good idea to distinguish request and response serializers. Should we have a separate module for request serializers instead of a module suffix? Perhaps something like `sentry.sentry_apps.api.parsers` for the request parsing serializers?

Reply:
that's a good idea. Prob should move the avatar request serializer into its own file while there 🫠